docs/tutorial/README.md

```diff
@@ -0,0 +1,215 @@
+see more configuration options.
+
+So after the service from the previous section is running, we can execute the
+`otelauto` that we previously downloaded with `go install`, as seen in the
+[Downloading](#downloading) section.
+
+We will configure it to instrument the executable that owns the port 8080, and
+to just printing the traces via standard output (not reporting to any OpenTelemetry
+endpoint).
+
+Remember that you need administrator access to run the instrumenter:
+
+```
+$ PRINT_TRACES=true OPEN_PORT=8080 sudo -E otelauto
+```
+
+You can now test the instrumented service from another terminal:
+
+```
+$ curl "http://localhost:8080/hello"        
+$ curl "http://localhost:8080/bye"
+```
+
+After some logs, the `otelauto` standard output should show the traces information
+of the above requests:
+
+```
+2023-04-19 13:49:04 (15.22ms[689.9µs]) 200 GET /hello [::1]->[localhost:8080] size:0B
+2023-04-19 13:49:07 (2.74ms[135.9µs]) 200 GET /bye [::1]->[localhost:8080] size:0B
+```
+
+The format is:
+
+```
+Request_time (response_duration) status_code http_method path source->destination request_size 
+```
+
+You can try to play with the `curl` command to see how it affects the traces.
+For example, the following request would send a 6-bytes POST request and the service will
+take 200ms to respond:
+
+```
+$ curl -X POST -d "abcdef" "http://localhost:8080/post?delay=200ms"
+```
+
+And the autoinstrumenter standard output will show:
+
+```
+2023-04-19 15:17:54 (210.91ms[203.28ms]) 200 POST /post [::1]->[localhost:8080] size:6B
+```
+
+## Sending data to Grafana Cloud
+
+Once we have verified that our application is correctly instrumented, we can modify the
+configuration of the Autoinstrumenter to send the data directly to Grafana Cloud.
+You can get a [Free Account in the Grafana site](https://grafana.com/pricing/).  
+
+> ⚠️ For simplicity, we will be [sending OpenTelemetry data directly to Grafana Cloud](https://grafana.com/docs/grafana-cloud/data-configuration/otlp/send-data-otlp/).
+However, this is a preview feature that is not available in all the regions. Another option
+is to [configure the Grafana Agent to process OpenTelemetry data and forward it to Grafana](https://grafana.com/docs/agent/latest/).
+
+In your Grafana Cloud Portal, click on the "Details" button in the "Grafana" box. Then
+get your Grafana Cloud Instance ID number and region, e.g.:
+
+![](./img/grafana-instance-id.png)
+
+Also make sure you create an API Key in the "Security → API Keys" section of your Grafana
+Cloud portal.
+
+Now you can configure the Autoinstrumenter via the following environment variables.
+
+```
+export GRAFANA_INSTANCE_ID="123123"
+export GRAFANA_API_KEY="abcabcabcabcabcabcab....bcabcabc="
+export GRAFANA_REGION="prod-eu-west-0"
+
+export OTEL_EXPORTER_OTLP_ENDPOINT="https://otlp-gateway-${GRAFANA_REGION}.grafana.net/otlp"
+export OTEL_EXPORTER_OTLP_PROTOCOL="http/protobuf"
```

Comment:
Do we need to add explicit support for protobuf for the protocol to work, or it's somehow handled automatically? I thought we might need to use the oltptracegrpc?

Reply:
Ahh good question, let me double-check and remove it.